code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -218,3 +218,12 @@
 	materials = list(MAT_BIOMASS = 300)
 	build_path = /obj/item/clothing/head/rice_hat
 	category = list("initial","Leather and Cloth")
+
+/datum/design/seasweed
+	name = "seaweed sheet"
```

Comment:
```suggestion
	name = "Seaweed Sheet"
```
It's not standardized over biogenerators but normally it's like this for designs.